doc/modules/impute.rst

```diff
@@ -87,37 +87,40 @@ string values or pandas categoricals when using the ``'most_frequent'`` or
      ['a' 'y']
      ['b' 'y']]
 
-.. _chained_imputer:
+.. _iterative_imputer:
 
 
 Multivariate feature imputation
 ===============================
 
-A more sophisticated approach is to use the :class:`ChainedImputer` class, which
-implements the imputation technique from MICE (Multivariate Imputation by
-Chained Equations). MICE models each feature with missing values as a function of
-other features, and uses that estimate for imputation. It does so in a round-robin
-fashion: at each step, a feature column is designated as output `y` and the other
-feature columns are treated as inputs `X`. A regressor is fit on `(X, y)` for known `y`.
-Then, the regressor is used to predict the unknown values of `y`. This is repeated
-for each feature in a chained fashion, and then is done for a number of imputation
-rounds. Here is an example snippet::
+A more sophisticated approach is to use the :class:`IterativeImputer` class,
+which models each feature with missing values as a function of other features,
+and uses that estimate for imputation. It does so in an iterated round-robin
+fashion: at each step, a feature column is designated as output `y` and the
+other feature columns are treated as inputs `X`. A regressor is fit on `(X, y)`
+for known `y`. Then, the regressor is used to predict the missing values of `y`.
+This is repeated for each feature in an iterative fashion, and then is done for a
+number of imputation rounds. The results of the final imputation round are
+returned. Our implementation was inspired by the R MICE package (Multivariate
```

Comment:
Refer to a reference note to the paper of MICE.

Reply:
What's the standard way to do this? Just put in `[1]`?

Reply:
We can make a section reference in sphinx with a number and then refer to the number as well.

Reply:
Sorry, I'm not very experienced in sphinx. Is there an example somewhere else in sklearn's documentation that I can copy?